R/PlainTextParam.R

```diff
@@ -0,0 +1,215 @@
+#' @title Store `MsExperiment` and `XcmsExperiment` object as a folder of plain
+#' text files
+#'
+#' @name PlainTextParam
+#'
+#' @export
+#'
+#' @family xcms result export formats.
+#'
+#' @description
+#' The `PlainTextParam` class and method enable users to save `MsExperiment` or
+#' `XcmsExperiment` objects as collections of plain text files in a specified
+#' folder.
+#'
+#' For an `MsExperiment` object, the exported files include:
+#'
+#' - The [`sampleData()`] stored as a text file named "sample_data.txt".
```

Comment:
can you maybe use `"` for the file names instead of only "? Or, alternatively * (the filenames would then be rendered in italics font)?

Reply:
With `"` I meant a " with the backticks - but maybe using italics for file names would be better?

Reply:
I like italics, i will go with that